.github/workflows/build.yml

```
@@ -63,6 +63,23 @@ jobs:
           pwd
           tilt ci
 
+      - name: Get Version
+        run: |
+          VER=$(cat VERSION)
+          echo "V_VERSION=$VER" >> $GITHUB_ENV
+
+      - name: Get Timestamp
+        run : |
+          TSTAMP=$(date +'%Y%m%d%H%M%S')
+          echo "V_TIMESTAMP=$TSTAMP" >> $GITHUB_ENV
+
+      - name: Save artifacts
+        uses: actions/upload-artifact@v3
```

maybe add a conditional so we only save artifacts for commits to main? 
```suggestion
        uses: actions/upload-artifact@v3
        if: (github.event_name != 'pull_request') && (github.event_name != 'schedule')
```
or if you want an artifact for every PR commit maybe add 
``` retention-days: 2``` 
so we arent saving a bunch of builds unncessarily

---

Excellent idea on restricting it to main only.  I like that better than expiring things on the PRs.